Rivet/Functions/Export-Migration.ps1

```diff
@@ -1163,12 +1163,18 @@ where
 -- SCHEMAS
 select 
     sys.schemas.name, 
-    sys.sysusers.name as owner 
+    sys.sysusers.name as owner,

```

Comment:
Do you need this join on sys.sysusers? We shouldn't be doing anything user-specific?

Reply:
Hmm this was already joining on the sys.sysusers table prior to my work. I can remove it if needed.